### ISSUE TITLE
Origin?

### ISSUE THREAD

**Issue by tsoj** [maxivolkov/kobol#1]
What is the connection between kobol and [StoneChess](https://github.com/Thibor/StoneChess)?

**Comment by tissatussa** [comment]
> What is the connection between kobol and StoneChess?

Why do you suspect a relation ? Known author ?

**Comment by AlexBrunetti** [comment]
Indeed, at first glance, there are many similarities in the source code of the two programs. However, you should consider that StoneChess is public domain...

**Comment by tissatussa** [comment]
> ..there are many similarities in the source code of the two programs..

i've looked into both code packages : indeed the similaries are obvious, many parts are almost exactly same. But the Kobol code is much cleaner and better, i instantly found a few grave errors in StoneChess, like double definitions, Classes in both (?!) .h and .cpp files and more, so i just decided to skip StoneChess.

**Comment by tsoj** [comment]
> Indeed, at first glance, there are many similarities in the source code of the two programs. However, you should consider that StoneChess is public domain...

Yes totally, just in case that this is a different author than the author of StoneChess, and that the origin of kobol is indeed StoneChess, then it would be good courtesy to mention the original author.
